src/core/components/hv-screen/index.js

```diff
@@ -15,16 +15,12 @@ import * as Events from 'hyperview/src/services/events';
 import * as Namespaces from 'hyperview/src/services/namespaces';
 import * as Render from 'hyperview/src/services/render';
 import * as Stylesheets from 'hyperview/src/services/stylesheets';
-import * as UrlService from 'hyperview/src/services/url';
-import * as Xml from 'hyperview/src/services/xml';
-import { ACTIONS, NAV_ACTIONS, UPDATE_ACTIONS } from 'hyperview/src/types';
-// eslint-disable-next-line instawork/import-services
-import Navigation, { ANCHOR_ID_SEPARATOR } from 'hyperview/src/services/navigation';
-import { createProps, createStyleProp, getElementByTimeoutId, getFormData, later, removeTimeoutId, setTimeoutId, shallowCloneToRoot } from 'hyperview/src/services';
-import { Linking } from 'react-native';
+import { createProps, createStyleProp, later } from 'hyperview/src/services';
 import LoadElementError from '../load-element-error';
 import LoadError from 'hyperview/src/core/components/load-error';
 import Loading from 'hyperview/src/core/components/loading';
+// eslint-disable-next-line instawork/import-services
```

Comment:
I'm curious if that's still needed

Reply:
It is. Without it, it wants me to import "Navigation" as a wildcard and that doesn't allow it to be instantiated with "new". Another area where we probably want to do some cleanup eventually and not export it as the default value.